src/main/java/org/takes/rq/form/RqFormBase.java

```diff
@@ -148,9 +143,9 @@ private static String decode(final CharSequence txt) {
      * @throws IOException If something fails reading or parsing body
      */
     private Map<String, List<String>> freshMap() throws IOException {
-        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        new RqPrint(this.req).printBody(baos);
-        final String body = new Utf8String(baos.toByteArray()).string();
+        final String body = new Utf8String(
+            new RqPrint(this.req).printBody()
+        ).string();
```

Comment:
@dalifreire looks like this part is also already done in https://github.com/yegor256/takes/blob/master/src/main/java/org/takes/rq/RqPrint.java#L64